client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -12,33 +12,33 @@ import VerticalNav from 'components/vertical-nav';
 import VerticalNavItem from 'components/vertical-nav/item';
 import Header from 'my-sites/upgrades/domain-management/components/header';
 import Main from 'components/main';
+import { get } from 'lodash';
 
-class Transfer extends React.Component {
-	render() {
-		const slug = this.props.selectedSite.slug,
-			domainName = this.props.selectedDomainName;
-		return (
+function Transfer( props ) {
+	const { selectedSite, selectedDomainName, translate } = props;
+	const slug = get( selectedSite, 'slug' );
+	const canTransferDomain = ! get( selectedSite, 'options.is_automated_transfer', false );
```

Comment:
if selectedSite isn't available I think this should be `canTransferDomain` should be false